site/app/templates/GlobalHeader.twig

```diff
@@ -223,6 +223,11 @@
                 {{ system_message }}
             </div>
             {% endif %}
+            <div class="system-message warning" id="socket-server-system-message" hidden>
+                Warning: Failed to connect to websocket server. This page will not dynamically update.
+                Manually reload this page to see updates.{% if sysadmin_email is defined %} Notify the
```

Comment:
I believe we want to use the [`empty`](https://twig.symfony.com/doc/3.x/tests/empty.html) test clause: `sysadmin_email is empty`

From my understanding `$this->core->getConfig()->getSysAdminEmail()` should always minimally return the empty string.